src/nvim/options.lua

```diff
@@ -7534,16 +7534,16 @@ return {
         horizontally centered in the window, as long as one does not come too
         close to the beginning of the line.
         After using the local value, go back the global value with one of
-        these two: >
+        these two: >vim
         	setlocal sidescrolloff<
         	setlocal sidescrolloff=-1
         <
         Example: Try this together with 'sidescroll' and 'listchars' as
         	 in the following example to never allow the cursor to move
-        	 onto the "extends" character: >
+        	 onto the "extends" character: >vim
 
-        	 :set nowrap sidescroll=1 listchars=extends:>,precedes:<
-        	 :set sidescrolloff=1
+        	 set nowrap sidescroll=0 listchars=extends:>,precedes:<
```

Comment:
I don't think the behavior of `sidescroll=0` is more desirable than `sidescroll=1`.

Reply:
That was accidental (probably unintended `ctrl-x`).